test/Core.Test/OrganizationFeatures/OrganizationSubscriptionUpdate/UpdateSecretsManagerSubscriptionCommandTests.cs

```diff
@@ -0,0 +1,631 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Enums;
+using Bit.Core.Exceptions;
+using Bit.Core.Models.Business;
+using Bit.Core.OrganizationFeatures.OrganizationSubscriptionUpdate;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Core.Tools.Enums;
+using Bit.Core.Tools.Models.Business;
+using Bit.Core.Tools.Services;
+using Bit.Core.Utilities;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Core.Test.OrganizationFeatures.OrganizationSubscriptionUpdate;
+
+[SutProviderCustomize]
+public class UpdateSecretsManagerSubscriptionCommandTests
+{
+    [Theory]
+    [BitAutoData]
+    public async Task UpdateSecretsManagerSubscription_NoOrganization_Throws(
+        Guid organizationId,
+        SutProvider<UpdateSecretsManagerSubscriptionCommand> sutProvider)
+    {
+        sutProvider.GetDependency<IOrganizationRepository>()
+            .GetByIdAsync(organizationId)
+            .Returns((Organization)null);
+
+        var organizationUpdate = new OrganizationUpdate
+        {
+            OrganizationId = organizationId,
+            MaxAutoscaleSeats = null,
+            SeatAdjustment = 0
+        };
+
+        await Assert.ThrowsAsync<NotFoundException>(
+            () => sutProvider.Sut.UpdateSecretsManagerSubscription(organizationUpdate));
```

Comment:
You should assert the content of the error message as well, to make sure you're getting the error message you think you are. `NotFoundException` could be thrown for multiple reasons.

Reply:
resolved